lessons/1.5-Functions/text.adoc

```diff
@@ -0,0 +1,19 @@
+= Functions
+
+Once your program grows larger, you will want to separate parts of code into
+named pieces. In C++, like in many other languages, you do this with
+*functions*.
+
+On line 8 we declare a function called `print_the_answer`. The body of the
+function starts with an opening curly brace and ends with a closing curly
+brace, just like with namespaces. That's because functions also start a
+_scope_, the difference, though, is that names from function's scope are not
+accessible outside of it.
+
+In order to _call_ a function, you use the _function call operator_, just like
+on line 17.
+
+You likely noticed, that the entry point for our program looks suspiciously
+like a function. That's because it *is* a function. It's a special function
+though: no other function in the program is allowed to have the name `main`
+and you are not allowed to call it yourself.
```

Comment:
Because, you are not. There's not much rationale given for that in the standard. Maybe we should pick some other phrase, that better conveys the idea? Perhaps "you are forbidden to call it yourself" is a better choice of words?

Reply:
> Perhaps "you are forbidden to call it yourself" is a better choice of words?

yes, currently the sentence feels building up to a _why_ reason and ends abruptly. Would it be a good idea to link [cppreference page](http://en.cppreference.com/w/cpp/language/main_function) saying you can read further about `main()` function here?